tests/test_harvest_bookmarks.py

```diff
@@ -1,189 +1,50 @@
-                                                            {"id": invoice_item_category_3["id"]}]}
+        set_up(cls, rec_count=3)
 
     @classmethod
     def tearDownClass(cls):
-        logging.info("Start Teardown")
-        for invoice in cls._teardown_delete['invoices']:
-            delete_stream('invoices', invoice['id'])
-        for expense in cls._teardown_delete['expenses']:
-            delete_stream('expenses', expense['id'])
-        # for project_user in cls._teardown_delete['project_users']:
-        #     delete_project_user(cls._teardown_delete['projects'][0]['id'], project_user['id'])
-        #     delete_project_user(cls._teardown_delete['projects'][0]['id'], project_user['id'])
-        for project in cls._teardown_delete['projects']:
-            delete_stream('projects', project['id'])
-        for role in cls._teardown_delete['roles']:
-            delete_stream('roles', role['id'])
-        for estimate in cls._teardown_delete['estimates']:
-            delete_stream('estimates', estimate['id'])
-        for category in cls._teardown_delete['estimate_item_categories']:
-            delete_stream('estimate_item_categories', category['id'])
-        for category in cls._teardown_delete['invoice_item_categories']:
-            delete_stream('invoice_item_categories', category['id'])
-        for category in cls._teardown_delete['expense_categories']:
-            delete_stream('expense_categories', category['id'])
-        # Deleteing Clients also deletes: contacts
-        for client in cls._teardown_delete['clients']: 
-            delete_stream('clients', client['id'])
-        # TODO cannot delete tasks b/c they have 'tracked time'
-        # TODO cannot delete time_entry
-
-        ##########  Uncomment for PURGE MODE ###############
-        # stream = ""
-        # logging.info("Comencing PURGE of stream: {}".format(stream))
-        # all_of_stream = get_all(stream)
-        # while all_of_stream:
-        #     for s in get_all(stream):
-        #         try:
-        #             delete_stream(stream, s['id'])
-        #         except AssertionError:
-        #             continue
-        #     all_of_stream = get_all(stream)
-        ####################################################
-        
+        tear_down(cls)
+
     def name(self):
         return "{}_bookmark_test".format(super().name())
 
+    def match_records(self, expected_record, actual_record):
+        """Method to check if all the fields of expected record is available in
+        actual_record.
+
+        Args:
+            expected_record (dict): Ecpected record (created/updated)
+            actual_record (dict): Actual record (Fetched from tap.)
+
+        Returns:
+            Boolean: Return true if expected record is actual record else False.
+        """
+        dict_items = actual_record.items()
+        for items in expected_record.items():
+            if items in dict_items:
+                return True
```

Comment:
This function will return True even if only the first value matches. This function will not check all the fields.

Reply:
Oh, I see now. Updated the logic to return false if any field is not available in actual record.